### PR TITLE
cmus recipe (x86)

### PR DESCRIPTION
**Pull request by miqlas**
vnzkrx3

Fixed, again :)

### CODE REVIEW

**Comment by waddlesplash** [comment]
In the future, please use push --force instead of creating a new PR.

File: media-sound/cmus/cmus-2.7.1.recipe [hunk 1]
```diff
@@ -0,0 +1,78 @@
+SUMMARY="A Console Based Audio Player"
+DESCRIPTION="
+cmus (C* MUsic Player) is a small, fast and powerful console audio player \ which supports most major audio formats. Various features include gapless \ playback, ReplayGain support, MP3 and Ogg streaming, live filtering, instant \ startup, customizable key-bindings, and vi-style default key-bindings."
```
Comment (waddlesplash) [inline media-sound/cmus/cmus-2.7.1.recipe]: Line is still not wrapped...?

**Comment by miqlas** [comment]
However i made it with push --force, it created a new commit for the fix. Should i delete everything, and start from scratch again?

**Comment by pulkomandy** [comment]
On Wed, Aug 26, 2015 at 10:25:07AM -0700, miqlas wrote:
> However i made it with push --force, it created a new commit for the fix. Should i delete everything, and start from scratch again?

You can use git rebase -i to rewrite your commit history.

git rebase -i HEAD~10 # Interactive rebase of last 10 commits

This will open a text editor with a list of the commits. You can replace
the "pick" word at the start of each line with another keyword. For
example, "fixup" will take the changes from a commit, and put them in
the previous one. "squash" will do the same and let you adjust the
commit message of the resulting commit. You can also completely remove
commits from the list, if they are unrelated. (the text file opened will
have instructions)

After saving and quitting the editor, git will perform the operations.
If it hits a problem, it will stop and ask you to fix things. Follow the
instructions and then use git rebase --continue until it is done. Or if
you get lost, use git rebase --abort to get back to the state you had
before starting the whole process.

Once you have done all this, use git pull --rebase to get the latest
commits from the haikuports tree, without making a merge commit (it is a
good idea to always update this way, at least when working with
haiku-related projects)

-- 
Adrien.

**Comment by miqlas** [comment]
Thanks, PulkoMandy, but i had no luck. I have now 4 commits.
Better erase everything...

**Pull request closed without merging**